test/component_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/cloudposse/test-helpers/pkg/atmos"
+	helper "github.com/cloudposse/test-helpers/pkg/atmos/component-helper"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+type ComponentSuite struct {
+	helper.TestSuite
+}
+
+func (s *ComponentSuite) TestBasic() {
+	const component = "ssm-parameter/basic"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
+
+	defer s.DestroyAtmosComponent(s.T(), component, stack, nil)
+	options, _ := s.DeployAtmosComponent(s.T(), component, stack, nil)
+	assert.NotNil(s.T(), options)
+
+	createdParam := aws.GetParameter(s.T(), awsRegion, "/dev/testing")
```

Comment:
We can not use `/dev/staging` for asserts. `/dev/staging` is shared resource between tests (opentofu and terraform) runs in parallel on CI. 

Could you please pass some additional random `ssm parameter` with input and verify it?